src/helm/benchmark/run_specs/lite_run_specs.py

```diff
@@ -305,3 +305,26 @@ def get_wmt_14_spec(language_pair: str, max_train_instances: int = 1) -> RunSpec
         metric_specs=get_open_ended_generation_metric_specs(),
         groups=["wmt_14"],
     )
+
+
+@run_spec_function("gpqa")
+def get_gpqa_spec(subset: str, method: str = ADAPT_MULTIPLE_CHOICE_JOINT) -> RunSpec:
```

Comment:
Delete `method` here because method is going to depend on whether we use CoT or not.